### PR TITLE
Issue#28224 fix a link in doc

### PR DESCRIPTION
**Pull request by nmuthusamy**
# What does this PR do?

This PR fixes a documentation link in pipeline docs. The link pointing to additional documentation for additional keyword arguments that pass along to the generate method of the model was incorrect which has been addressed in this PR.

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by amyeroberts** [comment]
cc @gante

**Comment by lappemic** [comment]
@nmuthusamy are you planning on further working on this?

File: src/transformers/pipelines/conversational.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ def __call__(self, conversations: Union[List[Dict], Conversation, List[Conversat
                 Whether or not to clean up the potential extra spaces in the text output.
             generate_kwargs:
                 Additional keyword arguments to pass along to the generate method of the model (see the generate method
-                corresponding to your framework [here](./model#generative-models)).
+                corresponding to your framework [here](https://huggingface.co/docs/transformers/generation_strategies#text-generation-strategies)).
```
[the remark this answers is not in the document]
Comment (gante) [reply src/transformers/pipelines/conversational.py:273]: Agreed with Steven!

**Comment by junhl** [comment]
I made the change through https://github.com/huggingface/transformers/pull/30948

**Comment by stevhliu** [comment]
Closed by #30948

**Pull request closed without merging**